src/com/lftechnology/java/training/srijan/basics/football1/PointsStructure.java

```diff
@@ -0,0 +1,148 @@
+
+	/**
+	 * <p>
+	 * This method helps to get Points from an Array
+	 * </p>
+	 * 
+	 * @param numberOfMatches
+	 *            {@link Integer}
+	 * @param fixturesWithScore
+	 *            {@link String}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void getPoints(int numberOfMatches, String[] fixturesWithScore) {
+		String result = "";
+		int stringHomePosition = 0;
+		int stringAwayPosition = 2;
+		for (int i = 0; i < numberOfMatches; i++) {
+			if (fixturesWithScore[stringHomePosition].equals("Manchester United")) {
+				result = determineResult(fixturesWithScore, stringHomePosition);
+			}
+
+			else if (fixturesWithScore[stringAwayPosition].equals("Manchester United")) {
+				result = resultDetermine(fixturesWithScore, stringAwayPosition);
+			}
+			stringHomePosition = stringHomePosition + 4;
+			stringAwayPosition = stringAwayPosition + 4;
+			displayPoints(result, numberOfMatches);
+
+		}
+	}
+
+	/**
+	 * <p>
+	 * Determines WIN, LOSE, or DRAW for Home Game
+	 * </p>
+	 * 
+	 * @param fixturesWithScore
+	 *            {@link String}
+	 * @param stringPosition
+	 *            {@link Integer}
+	 * @return WIN, LOSE or DRAW {@link String}
+	 */
+	public String determineResult(String[] fixturesWithScore, int stringPosition) {
+		if (Integer.parseInt(fixturesWithScore[stringPosition + 1]) > Integer
+				.parseInt(fixturesWithScore[stringPosition + 3])) {
+			return "WIN";
+		} else if (Integer.parseInt(fixturesWithScore[stringPosition + 1]) < Integer
+				.parseInt(fixturesWithScore[stringPosition + 3])) {
+			return "LOSE";
+		} else
+			return "DRAW";
+	}
+
+	/**
+	 * <p>
+	 * Determines WIN, LOSE or DRAW for Away Game
+	 * </p>
+	 * 
+	 * @param fixturesWithScore
+	 *            {@link String}
+	 * @param stringPosition
+	 *            {@link Integer}
+	 * @return WIN, LOSE, or DRAW {@link String}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public String resultDetermine(String[] fixturesWithScore, int stringPosition) {
+		if (Integer.parseInt(fixturesWithScore[stringPosition + 1]) > Integer
+				.parseInt(fixturesWithScore[stringPosition - 1])) {
+			return "WIN";
+		} else if (Integer.parseInt(fixturesWithScore[stringPosition + 1]) < Integer
+				.parseInt(fixturesWithScore[stringPosition - 1])) {
+			return "LOSE";
+		} else
+			return "DRAW";
+	}
+
+	/**
+	 * <p>
+	 * Displays points
+	 * </p>
+	 * <p>
+	 * Displays Number of Wins
+	 * </p>
+	 * <p>
+	 * Displays Number of Draws
+	 * </p>
+	 * <p>
+	 * Displays Number of Lose
+	 * </p>
+	 * 
+	 * @param result
+	 *            {@link String}
+	 * @param numberOfWeeks
+	 *            {@link Integer}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void displayPoints(String result, int numberOfWeeks) {
+		if (result.equals("WIN")) {
+			points = points + WIN;
+			wins++;
+		} else if (result.equals("LOSE")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "LOSE" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)